bee-ternary/src/num_conversions.rs

```diff
@@ -0,0 +1,204 @@
+// Copyright 2020 IOTA Stiftung
```

Comment:
Do you mind not adding this file yet ? It's only required by the transaction crate so I'd like to add it when time comes. Also I'd like to clean it a bit and add some more tests.

Reply:
Sure